python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -282,14 +262,18 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
     };
 
     // Apply the styling transformation
-    const information = rawInformation.map((val) => {
+    const transformInformation = (val: ActorInformation) => {
       const transform = transforms[val.label];
       if (transform !== undefined) {
         return transform(val);
       } else {
         return transforms["Identity"](val);
       }
-    });
+    };
+    const information = rawInformation.map(transformInformation);
+    const secondaryInformation = secondaryRawInformation.map(
+      transformInformation,
+    );
```

Comment:
Maybe we can extract the secondary information down here after transforming everything to avoid having to carry them through? That might simplify things a bit.

Reply:
we can't do it here because we lost the label in transformation, this prevent us from splitting